packages/datadog-plugin-express/test/index.spec.js

```diff
@@ -986,6 +985,207 @@ describe('Plugin', () => {
           })
         })
       })
+
+      describe('with configuration for middleware disabled', () => {
+        before(() => {
+          return agent.load(plugin, 'express', {
+            middleware: false
+          })
+        })
+
+        after(() => {
+          return agent.close()
+        })
+
+        beforeEach(() => {
+          express = require(`../../../versions/express@${version}`).get()
+        })
+
+        it('should not activate a scope per middleware', done => {
+          if (process.env.DD_CONTEXT_PROPAGATION === 'false') return done()
+
+          const app = express()
+
+          let span
+
+          app.use((req, res, next) => {
+            span = tracer.scope().active()
+
+            next()
+          })
+
+          app.get('/user', (req, res) => {
+            res.status(200).send()
+            try {
+              expect(tracer.scope().active()).to.equal(span)
+              done()
+            } catch (e) {
+              done(e)
+            }
+          })
+
+          getPort().then(port => {
+            appListener = app.listen(port, 'localhost', () => {
+              axios.get(`http://localhost:${port}/user`)
+                .catch(done)
+            })
+          })
+        })
+
+        it('should not activate a span for every middleware on a route with middleware disabled ', done => {
```

Comment:
If I understand the test properly, the title is misleading. Basically this is checking that middleware are not instrumented?

Reply:
yup that's correct, i've update the language here to be more appropriate (which is now in line with how koa/connect tests for this are worded)